src/stactools/modis/fragments/MOD10A1/061/raster-bands.json

```diff
@@ -0,0 +1,37 @@
+{
+    "NSDI_Snow_Cover": {
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "NSDI_Snow_Cover_Basic_QA": {
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "NDSI_Snow_Cover _Algorithm_Flags_QA": {
```

Comment:
```suggestion
    "NDSI_Snow_Cover_Algorithm_Flags_QA": {
```

Reply:
Removed space in name for "raster-bands.json" in MCD64A1, MOD10A1, MOD10A2, MOD13A1, MOD13Q1, MYD10A1, MYD10A2, MYD13A1, and MYD13Q1.